iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/methods/DeviceMethodTests.java

```diff
@@ -437,72 +230,6 @@ public void invokeMethodResponseTimeoutFailed() throws Exception
         assertTrue(buildExceptionMessage("Iot Hub did not throw the expected gateway timeout exception", testInstance.deviceTestManager.client), expectedExceptionCaught);
     }
 
-    @Test
-    @StandardTierHubOnlyTest
-    @ContinuousIntegrationTest
-    public void invokeMethodUnknownDeviceFailed() throws Exception
```

Comment:
Redundant test. Further down in this suite is the same test, but with additional error logic checking to differentiate between "Device not found" and "Module not found"